docs/en/observability/landing-page/data/recommended-data.json

```
@@ -0,0 +1,141 @@
+{
+  "title": "What do you want to observe?",
+  "tabIcons": true,
+  "thumbnailImages": true,
+  "tabs": [
+    {
+      "title": "APM",
+      "id": "apm",
+      "icon": "apmApp",
+      "items": [
+        {
+          "title": "Monitor your application performance",
+          "description": "Learn how to collect and visualize Application Performance Monitoring (APM) data with the Elastic Stack.",
+          "link": "ingest-traces.html",
+          "image": "ingest-traces.png"
+        },
+        {
+          "title": "Monitor your application logs",
+          "description": "Correlate your application logs and traces for increased visibility into blah blah",
```

@bmorelli25 is there something better (less repetitive) than this?

```suggestion
        {
          "title": "Monitor your application logs",
          "description": "Correlate your application logs and traces for increased visibility into your application.",
```

---

Edited ⬆️